
following error occured at Fri Mar 16 2018 10:44:47 GMT+0200 (Jerusalem Standard Time) : {"code":"ECONNREFUSED","errno":"ECONNREFUSED","syscall":"connect","address":"127.0.0.1","port":3306,"fatal":true}
following error occured at Fri Mar 16 2018 10:44:47 GMT+0200 (Jerusalem Standard Time) : "called by storeAPI.getStatistics => called by storeController.getStatistics => called by storeBL.getStatistics => ECONNREFUSED"
following error occured at Fri Mar 16 2018 10:44:47 GMT+0200 (Jerusalem Standard Time) : "called by storeAPI.getStatistics => called by storeController.getStatistics => unable to retrive statistics - please contact support center"
following error occured at Fri Mar 16 2018 10:44:47 GMT+0200 (Jerusalem Standard Time) : "called by storeAPI.getStatistics => called by storeController.getStatistics => called by storeBL.getStatistics => error in dal => Error: connect ECONNREFUSED 127.0.0.1:3306"
following error occured at Fri Mar 16 2018 10:44:47 GMT+0200 (Jerusalem Standard Time) : "called by storeAPI.getStatistics => called by storeController.getStatistics => unable to retrive statistics - please contact support center"
following error occured at Sat Mar 17 2018 20:55:42 GMT+0200 (Jerusalem Standard Time) : "called by customerAPI.getDuplicateCustomer => called by customerController.getDuplicateCustomer => called by customerBL.getDuplicateCustomer => error in dal => Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'"
following error occured at Sat Mar 17 2018 21:22:03 GMT+0200 (Jerusalem Standard Time) : "called by customerAPI.getDuplicateCustomer => called by customerController.getDuplicateCustomer => called by customerBL.getDuplicateCustomer => error in dal => Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'"
following error occured at Sat Mar 17 2018 21:49:52 GMT+0200 (Jerusalem Standard Time) : "called by customerAPI.getDuplicateCustomer => called by customerController.getDuplicateCustomer => called by customerBL.getDuplicateCustomer => error in dal => Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'"
following error occured at Sat Mar 17 2018 21:50:45 GMT+0200 (Jerusalem Standard Time) : "called by customerAPI.getDuplicateCustomer => called by customerController.getDuplicateCustomer => called by customerBL.getDuplicateCustomer => error in dal => Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'"
following error occured at Sat Mar 17 2018 21:53:04 GMT+0200 (Jerusalem Standard Time) : "called by customerAPI.getDuplicateCustomer => called by customerController.getDuplicateCustomer => called by customerBL.getDuplicateCustomer => error in dal => Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'field list'"
following error occured at Sat Mar 17 2018 22:01:02 GMT+0200 (Jerusalem Standard Time) : "called by productAPI.addUpdateProduct => called by productController.addUpdateProduct => called by productBL.addUpdateProduct => activity: updateProduct error in dal => Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 's bread',20,3.19)' at line 1"